tests/unit-tests/api/utilities/Tests_Functions.php

```diff
@@ -0,0 +1,497 @@
+<?php
```

Comment:
Domains are used in multiple tests throughout this file, IP address should be tested as well.